jekyll/_cci2/docker-layer-caching.md

```diff
@@ -66,52 +93,20 @@ DLC has **no** effect on Docker images used as build containers. That is, contai
 ## How DLC works
 {: #how-dlc-works }
 
-DLC caches your Docker image layers by creating an external volume and attaching it to the instances that execute the `machine` and Remote Docker jobs. The volume is attached in a way that makes Docker save the image layers on the attached volume. When the job finishes, the volume is disconnected and re-used in a future job. The layers downloaded in a previous job with DLC will be available in the next job that uses the same DLC volume.
-
-One DLC volume can only be attached to one `machine` or Remote Docker job at a time. If one DLC volume exists but two jobs that request DLC are launched, CircleCI will create a new DLC volume and attach it to the second job. From that point on, the project will have two DLC volumes associated with it. This applies to parallel jobs as well: if two `machine` jobs are run in parallel, they will get different DLC volumes.
+DLC caches your Docker image layers using a sparse file within the container/virtual machine used to run your job. The same applies to jobs that use parallelism. If two `machine` jobs are run in parallel, each VM will have a separate DLC cache.
 
-Depending on which jobs the volumes are used in, they might end up with different layers saved on them. For instance, the volumes that are used less frequently might have older layers saved on them.
+If, for example, the first run of your job takes over two minutes to build a Docker image, and nothing changes in the Dockerfile before the second run, the Docker file build steps happen instantly, in zero seconds.
 
-The DLC volumes are deleted after 3 days of not being used in a job.
+When none of the layers in the image change between job runs, DLC pulls the layers from the cache and reuses those instead of rebuilding the entire image.
 
-CircleCI will create a maximum of 30 DLC volumes per project, so a maximum of 30 concurrent `machine` or Remote Docker jobs per project can have access to DLC. This takes into account the parallelism of the jobs, so a maximum of 1 job with 30x parallelism will have access to DLC per project, or 2 jobs with 15x parallelism, and so on.
+If part of the Dockerfile changes (which changes part of the image), a subsequent run of the exact same job with the modified Dockerfile may still finish faster than rebuilding the entire image. This is because the cache can still be used for the first few steps that did not change in the Dockerfile. The steps that follow the change must be rerun because the Dockerfile change invalidates the cache for those layers.
 
-![Docker Layer Caching](/docs/assets/img/docs/dlc_cloud.png)
```

Comment:
I removed the image but atually it might be worth just updating "Volume" to "Cache" as it's still largely correct I think: https://circleci.com/docs/docker-layer-caching/#how-dlc-works

Reply:
@brentmmarks / @joshhsoj1902 is the image still accurate if we change "volume" to "cache"

Reply:
Yes and no. It will all be the same cache if all the jobs are started at the same time. Job run 3 in the docs is no longer accurate since we no longer store separate caches.

Reply:
i think let's just get rid of the image...